_sass/modules/_m-list-countries.scss

```diff
@@ -0,0 +1,132 @@
+.m-list {
+  .open-menu{
+    width: 10px;
+    float: right;
+    height: 10px;
+    margin-left: auto;
+    position: relative;
+    top: 10px;
+    fill:$color-1;
+  }
+  .lists-container {
+    &:after {
+      content: "";
+      display: table;
+      clear: both;
+    }
+    ul {
+      float: left;
+      display: block;
+      text-align: left;
+      width: 50%;
+      margin-top: rem(15px);
+      @media screen and (min-width: $screen-s) {
+        width: 33%;
+        text-align: center;
+        margin-top: rem(50px);
+        padding-left: rem(25px);
+        padding-right: rem(25px);
+      }
+      li {
+        line-height: 1.46;
+        @media screen and (min-width: $screen-s) {
+          line-height: 2.13;
+        }
+        a {
+          font-size: $font-size-xsmall;
+          color: $font-color-3;
+          @media screen and (min-width: $screen-s) {
+            font-size: $font-size-normal;
+          }
+        }
+      }
+    }
+  }
+  hr {
+    height: 1px;
+    width: 100%;
+    background-color: $color-3;
+    margin-top: rem(28px);
+    border: 0;
+    @media screen and (min-width: $screen-s) {
+      margin-top: rem(90px);
+    }
+  }
+  .c-section-title {
+    color: $font-color-6;
```

Comment:
c- prefix means this is a component that we re-use along the project. 
That is why it should live into the components folder.